sympy/integrals/tests/test_manual.py

```diff
@@ -561,3 +568,16 @@ def test_issue_23348():
     steps = integral_steps(tan(x), x)
     constant_times_step = steps.substep.substep
     assert constant_times_step.context == constant_times_step.constant * constant_times_step.other
+
+
+def test_hyperbolic():
+    assert_is_integral_of(sinh(2*x), cosh(2*x)/2)
+    assert_is_integral_of(x*cosh(x**2), sinh(x**2)/2)
+    assert_is_integral_of(tanh(x), log(cosh(x)))
+    assert_is_integral_of(coth(x), log(sinh(x)))
+    f, F = sech(x), 2*atan(tanh(x/2))
+    assert manualintegrate(f, x) == F
```

Comment:
Some of these integrals cannot be computed by the main `integrate` routine on master. Tests should be added for `integrate` to see that it can compute these. Also I would put these tests along with the main trig tests.